dashboard/test/ui/features/star_labs/aichat/chat.feature

```diff
@@ -0,0 +1,63 @@
+@no_mobile
+Feature: AI Chat
+
+  "AI Chat" is our lab that introduces students to generative AI
+  by allowing them to customize, then interact with large language models.
+
+  Background:
+    Given I create a levelbuilder named "Simone"
+    And I am on "http://studio.code.org/s/allthethings/lessons/47/levels/2"
+    And I click selector "#ui-close-dialog" once I see it
+    And I wait until element "#ui-close-dialog" is not visible
+    And I dismiss the teacher panel
+
+  # As of 9/4/24, cannot access SageMaker in Drone.
+  # More discussion in this Slack thread: https://codedotorg.slack.com/archives/C03CK49G9/p1725475362107969
+  @no_circle
+  Scenario: Making chat request gets response
+    When I press keys "Hello" for element "#uitest-chat-textarea"
+    And I wait until element "#uitest-chat-submit" is enabled
+    And I click selector "#uitest-chat-submit"
+    And I wait until element "[aria-label='AI bot']" is visible
+    Then element "[aria-label='AI bot']" has css property "background-color" equal to "rgb(224, 248, 249)"
+
+  Scenario: Editing system prompt produces success notification and saves
+    When I press keys "You are a silly chatbot" for element "#system-prompt"
```

Comment:
Maybe add check that element `"#system-prompt"` is blank  for readability? Wasn't sure what the initial text was until watching video.

Reply:
Sorry not sure I follow, is the initial text something that we care about?

Reply:
When I read the test steps, I was wondering about the case of the default system prompt already set by the levelbuilder to be 'You are a silly chatbot'. So wanted to confirm that the system prompt changed and was saved. Definitely a nit and no need to change! It just prompted me to go look at the video and see the system prompt default is set to `""` initially.